vignettes/programming_strategy.Rmd

```diff
@@ -612,22 +622,36 @@ The documentation will be updated at:
     @param old_param *Deprecated*, please use `new_param` instead.
     ```
 
-## Handling of Warning and Error
+## Handling of Messages, Warnings and Errors
 
 When a function or argument is deprecated, the function must be updated to issue
-a warning or error using `deprecate_warn()` and `deprecate_stop()`,
-respectively, as described above.
+a message, warning or error using `deprecate_inform()`, `deprecate_warn()` or 
+`deprecate_stop()`, respectively, as described above.
 
 There should be a test case added in the test file of the function that checks
-whether this warning/error is issued as appropriate when using the deprecated
+whether this message/warning/error is issued as appropriate when using the deprecated
 function or argument.
 
 ### Function
 
-In the initial release in which a function is deprecated the original function
-body must be replaced with a call to `deprecate_warn()` and subsequently all
+**Phase 1:** In the initial release in which a function is deprecated the original function
+body must be replaced with a call to `deprecate_inform()` and subsequently all
 arguments should be passed on to the new function.
 
+```r
+fun_xxx <- function(dataset, some_param, other_param) {
+  deprecate_inform("x.y.z", "fun_xxx()", "new_fun_xxx()")
```

Comment:
We need to specify the `details` argument if we want to include the date when the message will turn into a warning.

Reply:
umm... yeah that makes sense to not lock ourselves in and keep things simple!!

Reply:
I placed some generic message in there for us and linked the deprecation document